apps/protocol-api/src/prisma/resolvers/Attestation.ts

```diff
@@ -135,7 +138,10 @@ export class AttestationResolver {
         },
         contribution: {
           connect: {
-            on_chain_id: args.data.contributionOnChainId,
+            chain_id_on_chain_id: {
```

Comment:
@alexkeating it doesn't look like this lines up with the fields for `ContributionWhereUniqueInput`